src/observation/encodeMatcher/operators.js

```diff
@@ -24,6 +24,18 @@ const noNullComparisons: OperatorFunction => OperatorFunction = operator => (lef
 // Same as `a > b`, but `5 > undefined` is also true
 const weakGt = (left, right) => left > right || (left != null && right == null)
 
+const handleLikeValue = (v, defaultV) => typeof v === 'string' ? v : defaultV
+
+export const like: OperatorFunction = (left, right) => {
+  const leftV = handleLikeValue(left, '')
+  const rightV = handleLikeValue(right, '(?!)')
+  const regexp = `^${rightV}$`
+    .replace(/%/g, '.*')
+    .replace(/_/g, '.')
+
+  return new RegExp(regexp, 'i').test(leftV)
```

Comment:
half of this function is the same as in LokiJS worker — I think a util function (in utils/fp) that turns a LIKE query string into a RegExp would be appropriate